app/.editorconfig

```diff
@@ -0,0 +1,8 @@
+root = true
```

Comment:
is it for activation?

Reply:
This statement declares that the current directory is the root directory of the project. This prevents editors and IDEs to scan parent directories to find other `.editorconfig` files.

Reply:
i got it.